source/basic.tex

```diff
@@ -3350,7 +3350,7 @@
 \end{note}
 
 \pnum
-A program may end the lifetime of any object by reusing the storage
+A program may end the lifetime of any object by reusing or releasing the storage
 which the object occupies or by explicitly calling a destructor
 or pseudo-destructor\iref{expr.prim.id.dtor} for the object.
```

Comment:
This sounds like it's introducing a new normative rule; maybe an "as described above" would help? (Especially because storage reuse that creates a nested object doesn't end the lifetime of the enclosing object.)

Reply:
I agree, we are repeating the end-of-lifetime rules incompletely. Updated. What do you think?